doc-content/jbpm-docs/src/main/asciidoc/BAM/DashbuilderRuntime-exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -0,0 +1,193 @@
+[id='DashbuilderRuntime-exporting-importing-dashbuilder-data-proc_{context}']
+= Exporting and importing Dashbuilder data
```

Comment:
As we discussed, large concept and reference modules can contain sub-headings, but not procedure modules. Procs always must be only single-heading files. So need to change this one to a concept (don't forget to rename the anchor ID and file name), and title it as a noun phrase, something like "Dashbuilder data management" or something. Then need to break up all the proc sections that follow as individual proc files accordingly.

Reply:
As per our dicussion, I have converted this section as follows:
- Dashbuilder data management in {CENTRAL} (Concept module)
- Exporting Dashbuilder data (Procedure module)
- Importing Dashbuilder data (Procedure module)
- Deploying dashboards from {CENTRAL} on Dashbuilder Runtime (Procedure module)